### DISCUSSION TITLE
Multiple POI in offline search

### DISCUSSION THREAD

**Discussion by devemux86** [devemux86/cruiser#155] [Q&A] [answered]
Until now we could activate 1 POI file in offline search of points of interest.

In next Cruiser, multiple POI files can be activated / used in offline search.
(similar to how we load multiple map files)

<img src="https://user-images.githubusercontent.com/3484020/236681009-666923e3-3945-4f68-b8dc-dea0fa6f8f89.jpg" width="50%"/>

**Comment by devemux86** [comment, marked as answer]
Multiple POI files can be activated in offline search in [Cruiser 3.0.19](https://play.google.com/store/apps/details?id=gr.talent.cruiser).

**Reply by DolomitesTom** [reply]
perfect... will there be a new smaller france.poi?

**Reply by devemux86** [reply]
> will there be a new smaller france.poi?

Large POI will be updated, I am still testing, it takes a while with big countries.

Note that it is **better not to enable unnecessary POI** as it can slow down the offline search.
